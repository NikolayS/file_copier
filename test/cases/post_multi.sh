#!/bin/bash

# to run this scenario, COPIERSERVICE should be defined (`export COPIERSERVICE=https://your.copier.host.name`)

http --form POST "$COPIERSERVICE/?wh=1&md5=1" \
      cache-control:no-cache \
        content-type:application/x-www-form-urlencoded \
          postman-token:ecd808df-a9ac-9847-9b6a-6620f4ee5ca1 \
            src='https://www.google.ru/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png+http://www.underconsideration.com/brandnew/archives/facebook_2015_logo_detail.png+http://images2.fanpop.com/image/photos/10300000/apple-logo-apple-inc-10332560-299-313.jpg' -h | grep -v "Date:"

