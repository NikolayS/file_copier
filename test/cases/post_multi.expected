HTTP/1.1 200 OK
Server: nginx
Transfer-Encoding: chunked
X-File-Copier-Md5: 03f8f14ddbefc8449787c20fca6e9d58
X-File-Copier-Img-Width: 299
X-File-Copier-Img-Height: 519
X-File-Copier-Size: 119001
X-Location: /data/6f/74/f8/9b/6f74f89b9fdf288b2649d88a4657e0371ce90c319c3d73f53625a94d781c83c4.png
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5

